modules/labs-react/select/spec/Select.spec.tsx

```diff
@@ -49,6 +49,17 @@ describe('Select', () => {
     });
   });
 
+  // TODO: Replace ref test with verifyComponent after fixing Select to handle
+  // consumer-provided onKeyDown event handler. Currently fails the onKeyDown
+  // callback test in verifyComponent.
+  describe('when rendered with a ref', () => {
+    it('should set the ref to the button element', async () => {
+      const ref: React.RefObject<HTMLButtonElement> = React.createRef();
+      const {findByRole} = render(<Select onChange={cb} options={options} ref={ref} />);
+      expect(await findByRole(selectButtonRole)).toEqual(ref.current);
+    });
+  });
```

Comment:
The Labs Select doesn't handle a consumer-provided onKeyDown handler correctly, so it fails the callback test in `verifyComponent`. Addressing this in a later PR and testing ref forwarding separately in the meantime.

Reply:
This could probably be done pretty easily. Without using `mergeProps`, you instead do something like the following:

```tsx
const { onKeyDown } = props

const handlers = {
  onKeyDown(event) {
    onKeyDown?.(event)
    // other stuff
  }
}
```